### ISSUE TITLE
A file per handler?

### ISSUE THREAD

**Issue by SimonLab** [FAC-GM/app#176]
Should we create a file per handler to be able to create unit test?
con: create more file and it will be a bit more difficult to navigate in the code
pro: We can create unit tests for theses handlers just by require those files in our test

What do you think?

**Comment by heron2014** [comment]
@SimonLab I quite like idea to create a file per handler, maybe it will be more files but it should be more readable and understandable.

**Comment by nelsonic** [comment]
Agree. It's *generally* a good idea to have one handler per file to help with testability. It also greatly reduces merge conflicts and simplifies code review. Sure, you end up with more files,  but the project grows organically so it's relatively simple for new developers to learn/understand.
